helm/secrets-provider/values.schema.json

```diff
@@ -112,6 +112,16 @@
               "type": "string",
               "minLength": 1
             },
+            "timeout": {
+              "type": "number",
+              "minLength": 1,
+              "minimum": 5
```

Comment:
5 what? minutes? seconds? milliseconds?
Better call it `timeoutSec` to ensure readability.

Reply:
name changed to retryInterval because it wasn't really a timeout

Reply:
adding